src/css/_slider.scss

```diff
@@ -0,0 +1,77 @@
+// uncomment to visualize slider mechanism
+// input {
+//   border: solid;
+// }
+
+.population-slider {
+  position: absolute;
+  z-index: 1100;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  top: 10em;
+  left: 1em;
+}
+.min-max-slider {
+  width: 360px;
+  //   text-align: center;
+  height: 2em;
+  display: flex;
+  justify-content: space-between;
+  // margin-bottom: 50px;
+  // top: 10em;
+  // left: 1rem;
+}
+.min-max-slider > label {
+  display: none;
+}
+span.value {
+  height: 1.7em;
+  font-weight: bold;
+  display: inline-block;
+}
+.slider-legend {
+  display: flex;
+  justify-content: space-between;
+  width: 100%;
+  padding: 0;
+}
+.slider-legend span {
+  font-size: 0.5em;
+  opacity: 1;
+  flex: 1;
+  text-align: center;
+}
+.min-max-slider > input {
+  cursor: pointer;
+  position: absolute;
+}
+
+/* webkit specific styling */
```

Comment:
This comment is being applied to all of the rule set, but it only actually matters to line 52, so it should be moved closer. But, in reality, you can remove it -- the name `-webkit-appearance` already makes it obvious.

Btw, with Sass, you can use `//` for comments.